hw3/ex03-05/src/Search.jsx

```diff
@@ -0,0 +1,74 @@
+import "./styles.css";
+import * as React from "react";
+import Box from "@mui/material/Box";
+import TextField from "@mui/material/TextField";
+import axios from "axios";
+
+const url = "https://thronesapi.com/api/v2/Characters";
+
+export default function Search() {
+  const [characters, setCharacters] = React.useState([]);
+  const [matchedCharacter, setMatchedCharacter] = React.useState(null);
+
+  React.useEffect(() => {
```

Comment:
I would recommend using props to pass in the api data to each compoenent. that way you only have to make the call once